org.ardour.Ardour5.json

```diff
@@ -0,0 +1,452 @@
+{
+  "app-id": "org.ardour.Ardour5",
```

Comment:
I think upstream adds the version number for parallel installs and we should drop it.

Reply:
done